modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Numeric/Numeric.es.js

```diff
@@ -82,9 +82,19 @@ const Numeric = ({
 		if (inputRef.current) {
 			let newValue = value;
 
+			let decimalSymbol = symbols.decimalSymbol;
+
+			if (
+				newValue &&
+				!newValue.includes('.') &&
+				symbols.decimalSymbol != ','
+			) {
+				decimalSymbol = ',';
+			}
+
 			if (dataType === 'integer' && value) {
 				newValue = String(
-					Math.round(newValue.replace(symbols.decimalSymbol, '.'))
+					Math.round(newValue.replace(decimalSymbol, '.'))
```

Comment:
@alinedoleron, this behavior should be communicated on the Jira ticket, because the expected result mentioned there for converting `1.5` (decimal) is `1` (integer), although this logic will deliver `2` as the result.

Reply:
I'll check with the ticket reporter if it was a mistake, since I didn't change the way it was implemented.

Reply:
The reporter said it was just an example, not really based to the code. I'll edit the ticket.

Reply:
Great!